includes/Classifai/Services/LanguageProcessing.php

```diff
@@ -167,6 +186,24 @@ public function generate_post_excerpt( WP_REST_Request $request ) {
 		return rest_ensure_response( $provider->rest_endpoint_callback( $post_id, 'excerpt' ) );
 	}
 
+	/**
+	 * Generates text to speech for a post using REST.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return boolean
+	 */
+	public function synthesize_speech_from_text( $request ) {
+		$post_id           = $request->get_param( 'id' );
+		$save_post_handler = new SavePostHandler();
+		$attachment_id     = $save_post_handler->synthesize_speech( $post_id );
+
+		if ( is_wp_error( $attachment_id ) ) {
+			return false;
```

Comment:
Should we return any error message here? to inform users about text-to-speech generation is failed? what do you think?